modules/oe_corporate_blocks_demo/oe_corporate_blocks_demo.post_update.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Corporate blocks Demo post updates.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\block\Entity\Block;
+
+/**
+ * Delete the block siteswitcherblock.
+ */
+function oe_corporate_blocks_demo_post_update_00001_delete_the_siteswitcherblock() {
```

Comment:
Don't know does it make sense implement of post updates for *demo modules, because normally we don't keep this extensions turned on.
@voidtek what do you think?

Reply:
It's a good question. I will ask during the DS.

Reply:
According to the DS, I can keep it. 
But for the future, a post_update is not needed for a demo module.